ReactWindows/ReactNative.Shared/UIManager/BorderedCanvasManager.cs

```diff
@@ -12,16 +13,37 @@ namespace ReactNative.UIManager
     /// View parent manager for bordered canvases.
     /// </summary>
     public abstract class BorderedCanvasManager<TCanvas> : BorderedViewParentManager<TCanvas>
-        where TCanvas : Canvas
+        where TCanvas : BorderedCanvas
```

Comment:
Why can't you use `HasBorder` and `GetOrCreateBorder` to get the border reference and set the dimensions from the base class?

---
In reply to: [128647915](https://github.com/rozele/react-native-windows/pull/1#discussion_r128647915) [](ancestors = 128647915)